src/api/layers/layers.api.js

```diff
@@ -29,6 +29,8 @@ import log from '@/utils/logging'
 const generateClassForLayerConfig = (layerConfig, id, allOtherLayers, lang) => {
     let layer = undefined
     if (layerConfig) {
+        const maxResolution =
+            layerConfig.resolutions?.slice(-1)[0] ?? DEFAULT_MAX_GEOADMIN_RESOLUTION
```

Comment:
Do we currently have layer without max resolution ? Make it really sense to use a default value? 
Here a danger we might hide a bug in the backend, if currently all layer have a resolutions list, then we should either let the application crash in the case it is missing or remove the layer from the config with an error so we can quickly identify the issue.